src/Parser.js

```diff
@@ -443,9 +443,11 @@ export default class Parser {
         if (func === "\\begin") {
             // begin...end is similar to left...right
             const begin = this.parseGivenFunction(start);
+            // $FlowFixMe
```

Comment:
@marcianx we should be able to use `checkNodeType` or `assertNodeType` here to clean up these `$FlowFixMe`s right?